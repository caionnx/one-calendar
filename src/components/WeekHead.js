import React from 'react';

const WeekHead = () => (
  <tr className='WeekHead'>
    <th>Su</th>
    <th>M</th>
    <th>Tu</th>
    <th>W</th>
    <th>Th</th>
    <th>F</th>
    <th>Sa</th>
  </tr>
)

export default WeekHead;