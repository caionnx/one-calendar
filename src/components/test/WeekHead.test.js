import React from 'react';
import { shallow } from 'enzyme';

import WeekHead from '../WeekHead';

it('render WeekHead correctly', () => {
  const wrapper = shallow(<WeekHead />);

  expect(wrapper).toMatchSnapshot();
});