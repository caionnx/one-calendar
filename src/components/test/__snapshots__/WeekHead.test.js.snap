// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render WeekHead correctly 1`] = `
<tr
  className="WeekHead"
>
  <th>
    Su
  </th>
  <th>
    M
  </th>
  <th>
    Tu
  </th>
  <th>
    W
  </th>
  <th>
    Th
  </th>
  <th>
    F
  </th>
  <th>
    Sa
  </th>
</tr>
`;
